examples/monitor-with-thanos/tidb-monitor.yaml

```diff
@@ -4,37 +4,20 @@ metadata:
   name: basic
 spec:
   clusters:
-    - name: basic
+  - name: basic
+  thanos:
+    baseImage: thanosio/thanos
+    version: v0.17.2
   prometheus:
-    config:
-      configMapRef:
-        name: external-config
     baseImage: prom/prometheus
     version: v2.11.1
   grafana:
     baseImage: grafana/grafana
     version: 6.1.6
   initializer:
-    baseImage: pingcap/tidb-monitor-initializer
-    version: v3.0.13
+    baseImage: registry.cn-beijing.aliyuncs.com/tidb/tidb-monitor-initializer
```

Comment:
maybe better keep the docker hub url?

Reply:
@mikechengwei I think we should better keep the docker hub for stability 🤔

Reply:
I changed it yesterday. Do you have any other questions?

Reply:
sorry I saw the stale page, no problem then